.pfnci/script.sh

```diff
@@ -55,7 +58,7 @@ main() {
         docker_args+=(--env="SPREADSHEET_ID=${SPREADSHEET_ID}")
       fi
       run "${docker_args[@]}" \
-          "asia.gcr.io/pfn-public-ci/chainer-ci-prep.${TARGET}" \
+          "asia.gcr.io/pfn-public-ci/chainer-ci-prep.${TARGET}:${base_branch}" \
```

Comment:
Are these docker tag going to be added afterward?

Reply:
This docker image will be added via https://github.com/chainer/chainer/pull/8350/files#diff-8fe44c4bd2506d0a5db474fe567dd289R98.

Reply:
OK